client/scripts/views/components/sidebar/index.ts

```diff
@@ -112,6 +116,19 @@ export const OffchainNavigationModule: m.Component<{}, { dragulaInitialized: tru
           m.route.set(`/${app.activeId()}`);
         },
       }),
+      topics.map((t) => (
+        m(Button, {
+          fluid: true,
+          rounded: true,
+          active: onFeaturedDiscussionPage(m.route.get(), t.name),
+          label: t.name,
+          class: app.chain?.base === ChainBase.Substrate ? 'sub-button' : '',
```

Comment:
All of the topic buttons should have the class of 'sub-button', correct? This check that looks at Substrate ChainBase is not necessary here because it is not relevant to the sub-topic buttons.